code/game/objects/items/weapons/grenades/ghettobomb.dm

```diff
@@ -85,6 +114,39 @@
 /obj/item/weapon/grenade/iedcasing/prime() //Blowing that can up
 	update_mob()
 	explosion(get_turf(src.loc),-1,0,2)
+	if(shrapnel_list.len > 0)
+		var/i
+		var/atom/target
+		var/list/bodyparts = list("head","chest","groin","l_arm","r_arm","l_hand","r_hand","l_leg","r_leg","l_foot","r_foot")
+		for(i=1, i<=shrapnel_list.len, i++)
+			var/obj/item/shrapnel = shrapnel_list[i]
+			if(shrapnel.is_shrapnel)
+
+				var/amount = shrapnel.shrapnel_amount
+
+				while(amount > 0)
+					amount = amount - 1
+					target =pick(trange(6, src.loc))
```

Comment:
Not sure why this one got marked outdated, because it isn't.
I blame GitHub.
Oh well. Just make sure you do it, because this is probably the most important change I requested.